test/integration/spec/localtrackpublication.js

```diff
@@ -407,6 +405,55 @@ describe('LocalTrackPublication', function() {
       });
     });
 
+
+    [PRIORITY_LOW, PRIORITY_STANDARD, PRIORITY_HIGH].forEach(beforePriority => {
+      [PRIORITY_LOW, PRIORITY_STANDARD, PRIORITY_HIGH].forEach(afterPriority => {
+        if (beforePriority !== afterPriority) {
```

Comment:
We could remove this check here and have a test for updating the same priority as before. WDYT?

Reply:
if priority does not change then we won't receive `publishPriorityChanged`

Reply:
Yes, we could have that check too in the test?

Reply:
Thanks for the suggestion - I have updated the test to handle this case.